logger/logger.swift

```diff
@@ -1,10 +1,188 @@
 //
 //  logger.swift
-//  logger
+//  Test
 //
-//  Created by Daniel Brain on 10/12/20.
+//  Created by Daniel Brain on 10/11/20.
 //
 
-class logger {
+import Foundation
 
+let DEFAULT_PAYLOAD: Payload = [:]
+
+class Logger {
+    private let url: URL
+    private let prefix: String?
+    private let logLevel: LogLevel
+    private let flushInterval: Double
+    
```

Comment:
I think I have some reading to do on modularity in Swift. Can I create a separate file with constants and whatnot, but keep it private? Or does it have to be a private member of the `Logger` class for that to work?

Reply:
Depends on the use-case I would recommend keeping inside `Logger` class as this is the only place you will use this? Thoughts? Otherwise we create separate file for just constants? Doesn't fit use-case then :-)

Reply:
Also not having separate file will mean less memory footprint for this library :-)